### PR TITLE
Approve assessments and update task buttons

### LINKED ISSUE #7: Create functionality for approval of all answers
When the user approves the assessment of all answers, the assessment is locked and saved to localStorage.

### LINKED ISSUE #14: Use different colours on tasks on task page
There should be three different designs for the tasks based on the status of the assessment. These are:
- Finished
- Current
- Remaining

These boxes should route to the correct tasks.

### PR DESCRIPTION
**Pull request by sylvihuynh**
fix #7 fix #14 

When assessments are approved, they are saved to `localstorage` on key `{taskNumber_approved}`.
Task buttons are coloured based on the state of the assessments which are: approved (green), started (orange) and remaining (blue)

### CODE REVIEW

File: functions/helpFunctions.tsx [hunk 1]
```diff
@@ -47,25 +47,46 @@ export function insperaDataToTextboxObject(
 }
 
 export function saveAssessments(
-  assessments: AssessmentType[],
-  key: number
+  assessments: ApprovalType[],
```
Comment (amalieeh) [inline functions/helpFunctions.tsx:50]: Var dette den som lagret alt på slutten av assessment? Som vi ikke lenger brukte siden vi kaller på `saveBatch` på fullfør-knappen også?

File: functions/helpFunctions.tsx [hunk 2]
```diff
@@ -47,25 +47,46 @@ export function insperaDataToTextboxObject(
...
+  if (typeof window !== 'undefined') {
     // store array as a string
-    localStorage.setItem(key.toString(), JSON.stringify(assessments));
+    localStorage.setItem(key, JSON.stringify(assessments));
+  }
+}
+
+export function getAssessments(taskNumbers: number[]): number[][] {
+  //convert tasknumbers to string
+  const tasks = taskNumbers.map(String);
```
Comment (amalieeh) [inline functions/helpFunctions.tsx:62]: Why `String` in map? Is it stringifying the list of numbers?

File: functions/helpFunctions.tsx (hunk 1 above)
Comment (sylvihuynh) [reply functions/helpFunctions.tsx:50]: Correct 💯

File: functions/helpFunctions.tsx (hunk 2 above)
Comment (sylvihuynh) [reply functions/helpFunctions.tsx:62]: Correct ^💯

Comment (amalieeh) [reply functions/helpFunctions.tsx:62]: Damn, cool:)

File: functions/helpFunctions.tsx (hunk 1 above)
Comment (amalieeh) [reply functions/helpFunctions.tsx:50]: Smooth change 😎

**Review by amalieeh — APPROVED** [review]
Damn, our project is getting goood! Nice work! 🔥 🚒 
If you just resolve the conflicts first, and then it's good to go!
(And then I'll append the other PR once this is merged (its gonna be quite some conflicts))

**Pull request merged**